multiqc/modules/qc3C/qc3C.py

```diff
@@ -964,9 +964,22 @@ def _none_to(x, y):
                     )
                 )
             parsed = parsed[-1]
+            # Check if fields in json file are named as in the old version
+            if "unobs_fraction" not in parsed:
+                parsed["unobs_fraction"] = parsed["unobs_frac"]
+            if "vs_accepted" not in parsed["separation_bins"]:
+                parsed["separation_bins"]["vs_accepted"] = parsed["separation_bins"]["zvs_accepted"]
+
         except json.JSONDecodeError:
             log.warning("Could not parse qc3C JSON: '{}'".format(f["fn"]))
             return
+        except KeyError as ex:
+            log.error(
```

Comment:
Why / when does this happen? Does it need to be an error log - can it be fixed?

If it's a version thing for qc3c then I would typically make it a debug message.

I would also quite like to drop some of the other log messages in this module down to debug. It's a bit noisy currently..

Reply:
It happens if the input json file contain neither the old nor the new key, I think it should be an error message rather than a debug message therefore.

Reply:
Ok - any reason it's coming up so much from the files in https://github.com/ewels/MultiQC_TestData ?

Reply:
I hadn't realised that module was parsing two separate formats, of which one didn't have one of the field names I was checking for. This should be fixed with the latest commit.

Reply:
Great! Thanks for checking 👍🏻